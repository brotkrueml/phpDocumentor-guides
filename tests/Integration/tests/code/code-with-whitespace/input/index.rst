Code Block
==========

::

    Test code block with whitespace.
    Some more code
