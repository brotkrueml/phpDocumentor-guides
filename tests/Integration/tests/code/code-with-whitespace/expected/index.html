<!DOCTYPE html>
<html lang="en">
<head>
    <title>Code Block</title>

</head>
<body>
    <div class="section" id="code-block">
    <h1>Code Block</h1>

            <pre><code class="language-">Test code block with whitespace.
Some more code</code></pre>
    </div>

</body>
</html>
